### ISSUE TITLE
404 Not Found. But why? 

### ISSUE THREAD

**Issue by x86-app** [ziahamza/webui-aria2#498]
Hi. 
I created a systemctl service to run the server. And it always return '404 Not Found' in plain/text when I access http://(IP pointed to the server in LAN)/, and the request just headed on the root path. 
Service file and status are below. 

```
# File /lib/systemd/system/aria2-webui.service

[Unit]
Description=Aria2 WebUI Daemon
After=network.target

[Service]
Type=simple
User=root
Group=root
ExecStart=/usr/bin/node /usr/app/aria2-webui/node-server.js

[Install]
WantedBy=multi-user.target
```

```
# Status
root@Pi: /usr/app/aria2-webui master
 # systemctl status aria2-webui                                                       [22:16:09]
● aria2-webui.service - Aria2 WebUI Daemon
   Loaded: loaded (/lib/systemd/system/aria2-webui.service; disabled; vendor preset: enabled)
   Active: active (running) since Sat 2019-02-16 22:16:09 CST; 4s ago
 Main PID: 6544 (node)
   CGroup: /system.slice/aria2-webui.service
           └─6544 /usr/bin/node /usr/app/aria2-webui/node-server.js

2月 16 22:16:09 Pi systemd[1]: Started Aria2 WebUI Daemon.
2月 16 22:16:11 Pi node[6544]: WebUI Aria2 Server is running on http://localhost:8888
```

It seems all OK... But why there's a 404?

**Comment by x86-app** [comment]
Just tried with shell `/usr/bin/node /usr/app/aria2-webui/node-server.js` and it shows correctly.

**Comment by pratikborsadiya** [comment]
Try the fix from #477.

**Comment by anthonyraymond** [comment]
@x86-app problem solved?

**Comment by fluxtendu** [comment]
fix #477 works for me.
(on raspbian stretch)

**Comment by gitsang** [comment]
ExecStart=cd /usr/app/aria2-webui/ && /usr/bin/node node-server.js
